src/components/settings/unitList/CreateUnit.tsx

```diff
@@ -98,7 +98,6 @@ const CreateUnit: React.FC<CreateUnitProps> = ({ close, unit }) => {
             }))}
             InputProps={{ 'data-cy': 'unit-quantity' }}
             required
-            loading={loadingQuantities}
```

Comment:
Wasn't it better UX when you show the form and just loading on the autocomplete until quantities are loaded. Because like this the view jumps I guess from loading to form show after loading. Maybe there is a reason why you changed it but just wanted to be sure.

Reply:
It's because if units fail to load you end up with form that does not work. It's better to not to show broken form to a user at all, I though.

Reply:
I still think there could be some improvement but it is not so important at the moment.